index.js

```diff
@@ -65,6 +64,24 @@ NYC.prototype._createInstrumenter = function () {
   })
 }
 
+NYC.prototype._prepExcludePatterns = function (excludes) {
+  var directories = []
+  excludes = _.map(excludes, function (exclude) {
+    // Remove leading "current folder" prefix
+    if (_.startsWith(exclude, './')) {
+      exclude = exclude.slice(2)
+    }
+
+    // Allow gitignore style of directory exclusion
+    if (!_.endsWith(exclude, '/**')) {
```

Comment:
I didn't comment on the merit of it. Just that I would handle a trailing `/` better by first removing it.